hack/lib/version.sh

```diff
@@ -130,7 +130,9 @@ kube::version::ldflag() {
 kube::version::ldflags() {
   kube::version::get_version_vars
 
-  local -a ldflags=($(kube::version::ldflag "buildDate" "$(date -u +'%Y-%m-%dT%H:%M:%SZ')"))
+  local buildDate=
+  [[ -n ${SOURCE_DATE_EPOCH} ]] && buildDate="--date=@${SOURCE_DATE_EPOCH}"
```

Comment:
we have `nounset`, so this needs to be `[[ -n "${SOURCE_DATE_EPOCH-}" ]]`

otherwise LGTM